### PR TITLE
Fix the WASM example

### LINKED ISSUE #353: Wasm example not working
**Describe the bug**
I tried to follow the instructions to run the crates/matrix-sdk/examples/wasm_command_bot. Compiling works, but then I see this error:

`
ERROR in ./pkg/index_bg.wasm
Module parse failed: Unexpected section: 0x6c
..some stack trace...
ERROR in chunk 1
2912bd9d3988b701702b.module.wasm
/home/vsaase/src/matrix-rust-sdk/crates/matrix-sdk/examples/wasm_command_bot/pkg/index_bg.wasm
unexpected end
`

**Desktop (please complete the following information):**
Ubuntu 20.04 and OS X 11

### PR DESCRIPTION
**Pull request by poljar**
This closes #353.

### CODE REVIEW

**Review by jplatte — COMMENTED** [review]
I thought we already compile-checked the wasm example in CI? I think we should make sure to do so now, as it's very annoying to account for it in refactorings otherwise.

**Comment by poljar** [comment]
> I thought we already compile-checked the wasm example in CI? I think we should make sure to do so now, as it's very annoying to account for it in refactorings otherwise.

Yeah,  I was wondering the same how the error slipped through. I guess it isn't compiled with the rest of the examples because it's in a separate workspace.

**Comment by jplatte** [comment]
Makes sense, but then we should update xtask ci to take it into account.

File: xtask/src/ci.rs [hunk 1]
```diff
@@ -211,14 +212,29 @@ fn run_wasm_checks(cmd: Option<WasmFeatureSet>) -> Result<()> {
             .run()
     };
 
+    let test_command_bot = || {
+        let _p = pushd("crates/matrix-sdk/examples/wasm_command_bot");
+
+        cmd!("rustup run stable cargo clippy --target wasm32-unknown-unknown")
+            .args(["--", "-D", "warnings", "-A", "clippy::unused-unit"])
+            .run()
```
Comment (jplatte) [inline xtask/src/ci.rs:220]: Not sure whether it's better, but we could use cargo's `--manifest-path` instead of `pushd`.

**Review by jplatte — APPROVED** [review]

**Pull request merged**